infrastructure/firecracker/config.go

```diff
@@ -88,7 +89,11 @@ func WithMicroVM(vm *models.MicroVM) ConfigOption {
 			})
 		}
 
-		kernelArgs := vm.Spec.Kernel.CmdLine
+		kernelArgs := defaults.KernelCmdLine
```

Comment:
I think the provider should publish their default args (as a k:v)...related to comment above.